python/singa/autograd.py

```diff
@@ -440,6 +440,21 @@ def add_bias(x, b, axis=0):
     return AddBias(axis)(x, b)[0]
 
 
+class Sum(Operation):
+    def __init__(self):
```

Comment:
this is different to the definition of onnx sum operator.https://github.com/onnx/onnx/blob/master/docs/Operators.md#Sum

Reply:
yes, I just changed the code